packages/terra-functional-testing/src/commands/expect/toBeAccessible.js

```diff
@@ -8,7 +8,13 @@ const { runAxe } = require('../axe');
  */
 function toBeAccessible(_, options = {}) {
   const { result } = runAxe(options);
-  const { violations } = result;
+  const { incomplete, violations } = result;
+
+  // Rules that fail but are marked for review are returned in the incomplete array.
+  // https://github.com/dequelabs/axe-core/blob/develop/doc/API.md#results-object
+  if (incomplete && incomplete.length > 0) {
+    process.emit('terra:report:accessibility', { incomplete });
```

Comment:
Should we also add violations to the report's output? I could see two ways:
```
- one report, two sections
- two axe reports for incomplete and violations
``

Reply:
Violations are printed to the console when the expect fails; adding to the reporter also would print them again. We'd want to change how violations are printed if we wanted to pursue this, keeping in mind the expect would fail the test and print a message and then the reporter would print a message

Reply:
I don't think we need to add violations to the report. The failed test run should be indication enough. I see the report as valuable on non-failed test runs.